src/renderer/services/binance/service.ts

```diff
@@ -368,7 +370,15 @@ const freezeFee$: Observable<FeeRD> = FP.pipe(
   shareReplay(1)
 )
 
-const transaction = createTransactionService(clientState$)
+const wsTransfer$ = pipe(
+  address$,
+  switchMap(O.fold(() => Rx.EMPTY, subscribeTransfers)),
+  RxOperators.map(RD.success),
+  RxOperators.tap((state) => RD.isSuccess(state) && reloadBalances()),
```

Comment:
I'm not sure if we really want to reload balances with any incoming transfer event. Because it can be triggered by any transaction (even for buying chocolate in a candy store), but we don't need to refresh **all** balances if the user stays at the settings page etc. We should think about it again (no need to decide it with this PR).

Reply:
@veado u r definitely right